lib/matplotlib/_mathtext.py

```diff
@@ -2031,6 +2031,12 @@ def unknown_symbol(self, s, loc, toks):
         r'overrightarrow': r'\rightarrow',
         r'overleftarrow':  r'\leftarrow',
         r'mathring':       r'\circ',
+        r'r':              r'\circ',
+        r'=':              r'\combiningoverline',
+        r'u':              r'\combiningbreve',
```

Comment:
Removing this will solve an error when an incomplete `\underline` does not raise.